descriptors/ModuleDescriptor-template.json

```diff
@@ -905,6 +930,11 @@
       "displayName": "circulation - request queue for an item",
       "description": "get request queue for an item"
     },
+    {
+      "permissionName": "circulation.requests.hold-shelf-clearance-report.get",
+      "displayName": "circulation - request hold shelf clearance report CSV",
```

Comment:
This API does not produce the CSV. Please can the display name be updated to reflect that. And please remove the first pass terminology from the description

Reply:
Fixed